org.opent2t.sample.binaryswitch.superpopular/org.opent2t.sample.binaryswitch.superpopular.js

```diff
@@ -1,5 +1,5 @@
-var OcfSchemaReader = require("openT2T/schema/OcfSchemaReader");
+var OcfSchemaReader = require('openT2T/schema/OcfSchemaReader');
 
 module.exports = OcfSchemaReader.readThingSchemaFromFilesAsync(
-        require("path").join(__dirname, "org.opent2t.sample.binaryswitch.superpopular.raml")
+        require('path').join(__dirname, 'org.opent2t.sample.binaryswitch.superpopular.raml')
```

Comment:
All the new translators are trying to be consistent and use single quotes. I don't care either way, but 99% of the code was using single quotes except these js files so I made them consistent. #ByDesign.